telemetry/specs/INT.mdk

```diff
@@ -342,9 +346,12 @@ This may be addressed in the following ways -
 * It is recommended that the MTU of links between INT sources and sinks is
 configured to be higher than the MTU of preceding link MTUs (server/VM NIC MTUs)
 by an appropriate amount. Configuring an MTU differential of
-(Per-hop Metadata Length\*4\*INT Hop Count + 8), based on conservative values of
+[Per-hop Metadata Length\*4\*INT Hop Count + 12] bytes, based on conservative values of
```

Comment:
This is better, but it is still ignoring the tail header over TCP/UDP

Reply:
Correct, the magic number is 16